Multisig/UI/Settings/AppSettingsViewController/ChangeAppearanceViewController/ChangeDisplayModeTableViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ChangeDisplayModeTableViewController.swift
+//  Multisig
+//
+//  Created by Moaaz on 2/6/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ChangeDisplayModeTableViewController: UITableViewController {
+    private let items = [("Auto", UIUserInterfaceStyle.unspecified), ("Light", UIUserInterfaceStyle.light), ("Dark", UIUserInterfaceStyle.dark)]
```

Comment:
please add the type annotation with names for the tuple components to make the code more readable:

```
  private let items: [(name: String, style: UIUserInterfaceStyle)] = [("Auto", .unspecified), ... ]
```